src/main.cpp

```diff
@@ -26,3 +26,5 @@ int main() {
     QUILL_LOG_INFO(logger, "Current risk held is {}.", riskTracker.getRisk());
     return 0;
 }
+
+// adding this comment to do another pr for ethan
```

Comment:
why

Reply:
i was testing to make sure changes were showing up